spec/classes/debian_spec.rb

```diff
@@ -134,7 +135,22 @@
               ).with_content(
                 # This section varies for different releases
                 /\Unattended-Upgrade::Origins-Pattern\ {\n
-                \t"origin=Debian,archive=stable,label=Debian-Security";\n
+                \t"origin=Debian,codename=\${distro_codename},label=Debian-Security";\n
+                };/x
+              )
+            end
+          end
+        when 'buster'
+          context 'with defaults on Debian 10 Buster' do
+            it do
+              is_expected.to create_file(file_unattended).with(
+                owner: 'root',
+                group: 'root',
+                mode: '0644'
+              ).with_content(
+                # This section varies for different releases
+                /\Unattended-Upgrade::Origins-Pattern\ {\n
+                \t"origin=Debian,codename=\${distro_codename},label=Debian-Security";\n
                 };/x
```

Comment:
This is not correct, the default origins for buster are
```
"origin=Debian,codename=${distro_codename},label=Debian";
"origin=Debian,codename=${distro_codename},label=Debian-Security";
```

https://www.debian.org/releases/buster/amd64/release-notes/ch-whats-new.en.html#unattended-upgrades

Reply:
With those changes the PR works for me

Reply:
Thank you for your review. I thought this was some rule the module author implemented for unknown reasons, but with your provided link it makes much more sense. I'll happily make the changes.